demos/common/python/models/bert.py

```diff
@@ -0,0 +1,196 @@
+        pad_len = self.max_length - len(input_ids)
+        if pad_len < 0:
+            raise ValueError("The input request is longer than max number of tokens ({})"
+                             " processed by model".format(self.max_length))
+        input_ids += self.token_pad * pad_len
+        token_type_ids += self.token_pad * pad_len
+        attention_mask += self.token_pad * pad_len
+        return pad_len
+
+    def create_input_array(self, input_ids, attention_mask, token_type_ids):
+        inputs = {
+            self.input_names[0]: np.array([input_ids], dtype=np.int32),
+            self.input_names[1]: np.array([attention_mask], dtype=np.int32),
+            self.input_names[2]: np.array([token_type_ids], dtype=np.int32),
+        }
+        if len(self.input_names) > 3:
+            inputs[self.input_names[3]] = np.arange(len(input_ids), dtype=np.int32)[None, :]
+
+        return inputs
+
+    def reshape(self, new_length):
+        new_shapes = {}
+        for input_name, input_info in self.net.input_info.items():
+            new_shapes[input_name] = [1, new_length]
+        try:
+            default_input_shape = input_info.input_data.shape
+            self.net.reshape(new_shapes)
+            self.logger.debug("\tReshape model from {} to {}".format(default_input_shape, new_shapes[input_name]))
+            self.max_length = new_length
+        except RuntimeError:
+            raise RuntimeError("Failed to reshape the model")
+
+
+class NamedEntityRecognition(Bert):
+
+    def __init__(self, ie, model_path, vocab, input_names):
+        super().__init__(ie, model_path, vocab)
+
+        self.check_input_output_names(input_names)
+        self.output_names = list(self.net.outputs)
+        if len(self.output_names) != 1:
+            raise RuntimeError("The demo supports topologies only with 1 output")
+
+    def form_request(self, inputs):
+        c_tokens_id = inputs
+        input_ids = self.token_cls + c_tokens_id + self.token_sep
+        attention_mask = [1] * len(input_ids)
+        token_type_ids = [0] * len(input_ids)
+        return input_ids, attention_mask, token_type_ids
+
+    def postprocess(self, outputs, meta):
+        output = outputs[self.output_names[0]]
+        output = np.exp(output[0])
+        score = output / output.sum(axis=-1, keepdims=True)
+        labels_id = score.argmax(-1)
+
+        filtered_labels_id = [
+            (i, label_i) for i, label_i in enumerate(labels_id)
+            if label_i != 0 and 0 < i < self.max_length - meta['pad_len']
+        ]
+        return score, filtered_labels_id
+
+
+class BertEmbedding(Bert):
+
+    def __init__(self, ie, model_path, vocab, input_names):
+        super().__init__(ie, model_path, vocab)
+
+        self.check_input_output_names(input_names)
+        self.output_names = list(self.net.outputs)
+        if len(self.output_names) != 1:
+            raise RuntimeError("The demo supports topologies only with 1 output")
```

Comment:
```suggestion
            raise RuntimeError("The BertEmbedding model wrapper supports only 1 output")
```

Reply:
The messages of `RuntimeError` were updated.